### PR TITLE
Add CoreCLR projs to xplat sln

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

**Comment by jaredpar** [comment]
@dotnet-bot  test this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
FINALLY A PASSING BUILD

@jaredpar

**Comment by jaredpar** [comment]
@agocke everyone gets lucky now and again.  We need consistency :smile:

@dotnet-bot test this please

**Comment by jaredpar** [comment]
@dotnet-bot test this please

**Comment by agocke** [comment]
@jaredpar the one before failed because WINDOWS failed :<

**Comment by jaredpar** [comment]
@dotnet-bot test this please

**Comment by agocke** [comment]
test this please

**Comment by agocke** [comment]
test this please

**Comment by jaredpar** [comment]
Sorry. Was scrolling on my phone and tapped close.

**Comment by agocke** [comment]
Retest this please

**Comment by agocke** [comment]
Test this please

**Comment by agocke** [comment]
It looks like about 50%, but windows is failing a lot of these.

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

File: cibuild.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ XUNIT_VERSION=2.0.0-alpha-build2576
 BUILD_CONFIGURATION=Debug
 OS_NAME=$(uname -s)
 USE_CACHE=true
-MONO_ARGS='--runtime=v4.0.30319 --gc=boehm --debug=mdb-optimizations --attach=disable'
+MONO_ENV_OPTIONS='--runtime=v4.0.30319 --gc=boehm --debug=mdb-optimizations --attach=disable'
```
Comment (jaredpar) [inline cibuild.sh]: IIRC: that doesn't set an environment variable, it sets a bash variable.  Need to use `export` to set an ENV variable.

**Pull request closed without merging**